### ISSUE TITLE
FormerlySerializedAsAttribute incorrectly marked as redundant when used with SerializeReference

### ISSUE THREAD

**Issue by LarsBlaabjerg** [JetBrains/resharper-unity#2402]
The FormerlySerializedAsAttribute is incorrectly marked as redundant when used with SerializeReference.

![image](https://user-images.githubusercontent.com/4222097/234793061-fb864658-6555-460a-9ff6-f2ce8c669236.png)

Easy to reproduce, so I'll leave it at that.

**Comment by citizenmatt** [comment]
Could you try Rider 2023.1, please? There has been considerable work with regards to `[SerializeReference]` in this release, and it appears to be working correctly.

<img width="419" alt="Screenshot 2023-04-27 at 9 15 56 am" src="https://user-images.githubusercontent.com/222659/234802050-73e057c7-bfd5-4016-a8ce-9753298f6d4d.png">

**Comment by LarsBlaabjerg** [comment]
Yeah. Sorry. I had just updated Rider to make sure I had the latest version. I didn't expect to have to do it again. In JetBrains Rider 2023.1.1, it does work correctly. Thanks, and sorry for wasting your time :-)
